en/docs/develop/customizing-identity-server-uis.md

```diff
@@ -211,6 +211,27 @@ Add the `product-footer.jsp` and `header.jsp` files to the extensions folder as
 
     1. Open the `product-footer.jsp` file in the `<IS_HOME>/repository/deployment/server/webapps/authenticationendpoint/extensions` folder.
 
+    2. For example, to add the company name, you can replace the content of the `<footer>` tag with the following code.
+    ```html
+        <footer class="footer">
+            <div class="container-fluid">
+                <p>XYZ Company | &copy;
+                    <script>document.write(new Date().getFullYear());</script>
+                    <a href="<%=AuthenticationEndpointUtil.i18n(resourceBundle, "business.homepage")%>"
```

Comment:
@sherenewso2 Do we have a value define in i18n for this key by default?

Reply:
It is defined as http://wso2.com/ in <IS_HOME>/repository\deployment\server\webapps\authenticationendpoint\WEB-INF\classes\org\wso2\carbon\identity\application\authentication\endpoint\i18n\Resources.properties

It would be nice if it was https://wso2.com/